integration_tests/iceberg-sink/docker-compose.yml

```diff
@@ -84,6 +114,47 @@ services:
       - /datagen --mode clickstream --qps 1 mysql --user mysqluser --password mysqlpw --host mysql --port 3306 --db mydb
     container_name: datagen
     restart: on-failure
+  postgres:
```

Comment:
Which components are depending on `postgres` and `redis`? We'd better add dependencies on them for the components depending on them.